pages/candy-machines/[candy_machine_ID].tsx

```diff
@@ -39,6 +39,7 @@ const CandyMachine: NextPage = () => {
 
     const fetchNfts = async () => {
         if (!connection) return
+        setNftRecoilState([])
```

Comment:
why this update? was it wrong before?

Reply:
Yes. I had an issue that I switched from 1 cm to another one and the NFTs from the previous CM were showing.